packages/flutter_version_resolver/README.md

```diff
@@ -0,0 +1,4 @@
+# Flutter Version Resolver
```

Comment:
We should also include this new package in the README

Reply:
Done